storage/engine/rocksdb.go

```diff
@@ -417,6 +417,10 @@ func (r *RocksDB) Commit() error {
 	return nil
 }
 
+// Defer is a noop for RocksDB engine.
+func (r *RocksDB) Defer(func()) {
```

Comment:
Should this method and `rocksDBSnapshot.Defer` panic instead of silently doing nothing?